src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2150,7 +2150,6 @@ internal enum ErrorCode
         WRN_RefReturnOnlyParameter2 = 9095,
         ERR_RefAssignValEscapeWider = 9096,
         WRN_RefAssignValEscapeWider = 9097,
-
         #endregion
```

Comment:
> #endregion [](http://example.com/codeflow?start=7&length=11)

It looks like an empty line was removed above this directive. Consider restoring it.

Reply:
restored